SIL.ReleaseTasks/SetReleaseNotesProperty.cs

```diff
@@ -71,6 +73,10 @@ public override bool Execute()
 
 		private string ConvertLatestChangelog(int level, int skipUntilLevel = -1)
 		{
+			string urlRegex = @"(\[([^]]+)\]: (http|https|ftp|)://[0-9a-zA-Z]([-.\w]*[0-9a-zA-Z])*(:(0-9)*)*(/?)" +
+					@"([-a-zA-Z0-9.?,'/\+&%$#_]*)?([-a-zA-Z0-9?,'/+&%$#_]+))+";
```

Comment:
We don't need to extract the version number in [] here, so the group inside of `\[...\]` is unnecessary. Also, we don't need the outer group. And we don't really care if it's a valid URL as long as it starts with `https://` or similar. All this to say this can probably be simplified to:

```
			string urlRegex = @"\[[^]]+\]: (http|https|ftp|)://.+";
```